README.md

```diff
@@ -36,6 +36,7 @@ If you are interested in GraphQL in general, check out [awesome-graphql](https:/
 | API | Description | Graph*i*QL | Docs/Repo
 | --- | ----------- | :--------: | :-: |
 | Contentful | "CMS as a Service". Graph*i*QL demo allows to query a simple blog, but the library itself generates a schema automatically for any content model you store in Contentful. | [Try it!](https://cf-graphql-demo.now.sh/) | [Repo](https://github.com/contentful-labs/cf-graphql)
+| Discord | WidgetBot widgets for Discord, powered by GraphQL 	| [Try it!](https://widgetbot.io/api/graphql/) | [Repo](https://github.com/widgetbot-io)
```

Comment:
@samdenty99 Thanks, now makes a lot of sense 👍

> [Repo](https://github.com/widgetbot-io)

It's your entire GitHub organization so as a user I would be very confused since I expected to see the source code of your proxy.
 
Do you have any url to source code of your proxy?
Or any docs, it could be anything e.g. blog article?
If not just leave this column empty.